compiler/ballerina-lang/src/test/java/org/ballerinalang/toml/ManifestProcessorTest.java

```diff
@@ -131,9 +131,20 @@ public void testSingleDependencies() throws TomlException, IOException {
         Path tmpDir = Files.createTempDirectory("manifest-test-");
         Path baloPath = tmpDir.resolve("string_utils.balo");
         Files.createFile(baloPath);
-        
-        Manifest manifest = ManifestProcessor.parseTomlContentFromString(this.validProjectBlock + "[dependencies] \n " +
-                "string-utils = {path = \"" + baloPath + "\", version = \"1.1.5\"} \n");
+
+        String baloPathStr = baloPath.toString();
+        String tomlData;
+
+        if (baloPathStr.contains("/")) {
+            tomlData = this.validProjectBlock + "[dependencies] \n " +
```

Comment:
We can simplify this logic. This line is redundant.

Reply:
simplified